src/containers/CreateTaskRun/CreateTaskRun.js

```diff
@@ -450,61 +406,57 @@ class CreateTaskRun extends React.Component {
               lowContrast
             />
           )}
-          {!taskInfoDisabled && (
-            <FormGroup legendText="">
-              <Dropdown
-                id="create-taskrun--kind-dropdown"
-                titleText="Kind"
-                label=""
-                initialSelectedItem={{ id: 'task', text: 'Task' }}
-                items={[
-                  { id: 'task', text: 'Task' },
-                  { id: 'clustertask', text: 'ClusterTask' }
-                ]}
-                itemToString={itemToString}
-                onChange={this.handleKindChange}
+          <FormGroup legendText="">
+            <Dropdown
+              id="create-taskrun--kind-dropdown"
+              titleText="Kind"
+              label=""
+              initialSelectedItem={{ id: 'task', text: 'Task' }}
+              items={[
+                { id: 'task', text: 'Task' },
+                { id: 'clustertask', text: 'ClusterTask' }
+              ]}
+              itemToString={itemToString}
+              onChange={this.handleKindChange}
+            />
+            <NamespacesDropdown
+              id="create-taskrun--namespaces-dropdown"
+              invalid={validationError && !namespace}
+              invalidText={intl.formatMessage({
+                id: 'dashboard.createTaskRun.invalidNamespace',
+                defaultMessage: 'Namespace cannot be empty'
+              })}
+              selectedItem={namespace ? { id: namespace, text: namespace } : ''}
+              onChange={this.handleNamespaceChange}
+            />
+            {this.state.kind === 'Task' && (
```

Comment:
pull out this status.kind and error fields on line 345 ?

Reply:
yep, updated 👍